attributes/network/9_account_features/cddl/1_account_multisig.cddl

```diff
@@ -0,0 +1,212 @@
+; Account multisig feature API.
+; tag::types[]
+account-role /= "canMultisigSubmit"
+             /  "canMultisigApprove"
+
+account-multisig-token = bstr
+
+account-multisig-transaction = ledger-transaction-info
```

Comment:
I **attempted** to make it better locally but I would need your `cddl-fu` to confirm that I didn't do anything stupid.